### ISSUE TITLE
`export as namespace` not yet supported

### ISSUE THREAD

**Issue by GordonSmith** [Swatinem/rollup-plugin-dts#73]
The following declarion:
```
export as namespace dagre;
```

Causes the following error:
```
[!] (plugin dts) Error: Could not load C:\Users\gordon\git\hpcc-js\node_modules\@types\dagre\index.d.ts (imported by C:\Users\gordon\git\hpcc-js\packages\graph\types\graph2\layouts\dagre.d.ts): Syntax not yet supported
```

**Comment by Swatinem** [comment]
Hm, I figured out what this syntax is actually for:

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#support-for-umd-module-definitions

So its about UMD exports, which I guess we can support at some point when actually *outputting* stuff…

But I’m a bit confused why this actually tries to load your external `@types`. Those should be flagged as `external` by default.

**Comment by Swatinem** [comment]
I released `1.2.1` where I just ignore that statement when importing files. The other part is to actually *generate* that statement for our own bundles if requested. But that is for the future.
